src/src/AdminBundle/Entity/EventLanguages.php

```diff
@@ -67,11 +67,11 @@ public function setLanguage($language)
      * @ORM\ManyToOne(targetEntity="Event", inversedBy="languages")
      * @ORM\JoinColumn(name="event_id", referencedColumnName="id")
      */
-    private $event;
+    private $events;
```

Comment:
Tabuľka `event_languages` ti robí väzbu many to many. Ti znamená ze jeden riadok v tejto tabuľke ti naviaze jeden jazyk na jeden event. Premenovala by som to na `event` a `language`. A nezabudni pomeniť názvy - gettery a settery.